src/MoBi.UI/Views/BaseStartValuesView.cs

```diff
@@ -66,30 +64,6 @@ protected BaseStartValuesView(ValueOriginBinder<TStartValue> valueOriginBinder)
          _screenBinder = new ScreenBinder<IStartValuesPresenter<TStartValue>>();
       }
 
-      public override void InitializeResources()
-      {
-         base.InitializeResources();
-
-         scaleApplyToSelectionLayoutItem(layoutItemIsPresent);
```

Comment:
@Yuri05 Trying to do too much sometimes..., Here it seems to work when we don't try to size ourselves
![image](https://user-images.githubusercontent.com/1041237/177057303-5aeee0d6-0d46-477f-b1a2-ece30931ed4c.png)